elements/pl-rich-text-editor/pl-rich-text-editor.py

```diff
@@ -68,7 +68,7 @@ def render(element_html, data):
             'editor_uuid': uuid,
             'question': data['panel'] == 'question',
             'submission': data['panel'] == 'submission',
-            'read_only': 'true' if data['panel'] == 'submission' else 'false'
+            'read_only': 'true' if (data['panel'] == 'submission' or not data['editable']) else 'false'
```

Comment:
Is checking the panel still necessary with the addition of the other condition?

Reply:
Yes, it is needed. `data['editable']` can still be true in the submission panel. In essence, `data['editable']` indicates that the question still allows more submissions, even if the panel being rendered is not the question panel itself.